kamon-play-2.6.x/src/main/scala/kamon/play/instrumentation/instrumentation.scala

```diff
@@ -34,10 +36,10 @@ package object instrumentation {
 
   private def readOnlyTextMapFromHeaders(request: HttpRequest): TextMap = new TextMap {
 
-    private val headersMap = request.headers.map { h => h.name() -> h.value() }.toMap
```

Comment:
There are quite a few changes for Play 2.6 and this is no longer usable. I'm thinking that instead of having `GenericRequest` expose a Map for the headers it should just have a get.

Reply:
trying to introduce tests with both akka-http and netty-server and it seems there are some deps issues. will have to continue tomorrow...